gulpfile.js

```diff
@@ -202,6 +203,11 @@ gulp.task('imagemin', function () {
     .pipe(gulp.dest('public/dist/img'));
 });
 
+// Make sure upload directory exists
+gulp.task('mkdir:upload', function () {
```

Comment:
I know, but Grunt's convention so to use different subtasks, where was can just create a proper task name in Gulp. I feel like we shouldn't keep old conventions.